packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -48,9 +49,11 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
 
   logger.info("Lodestar", {version: lodestarGitData.version, network: args.network});
 
+  const dbMetrics = createDbMetrics();
```

Comment:
Metrics are disabled by default, here seems that they will always exists even if the user doesn't use --metrics.enable

Reply:
you are right, will fix